test/events/failIssueRes.json

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
✨ Thank you both for this suggestion! I'm not super familiar with all the best testing practices, but this definitely makes sense and I did update those changes.